website/source/docs/job-specification/check_restart.html.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: "docs"
+page_title: "check_restart Stanza - Job Specification"
+sidebar_current: "docs-job-specification-check_restart"
+description: |-
+  The "check_restart" stanza instructs Nomad when to restart tasks with
+  unhealthy service checks.
+---
+
+# `check_restart` Stanza
+
+<table class="table table-bordered table-striped">
+  <tr>
+    <th width="120">Placement</th>
+    <td>
+      <code>job -> group -> task -> service -> **check_restart**</code>
+    </td>
+  </tr>
+  <tr>
+    <th width="120">Placement</th>
+    <td>
+      <code>job -> group -> task -> service -> check -> **check_restart**</code>
+    </td>
+  </tr>
+</table>
+
+As of Nomad 0.7 the `check_restart` stanza instructs Nomad when to restart
```

Comment:
Mention the merge behavior

Reply:
Added https://github.com/hashicorp/nomad/pull/3105/commits/10ae18c7eb434af5f57b21a3db3d2acc11328433